components/Map/Map.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
+
+import MapView, { Marker } from 'react-native-maps';
+import { StyleSheet, Text, View, Button, TextInput } from 'react-native';
+
+export default function Map() {
+
+let events = [
+    {
+      id: 1,
+      date: 20200407,
+      time: 1700,
+      unix_time: 1586278800,
+      activity: 'Volleyball',
+      location: 'Wash Park',
+      lat: 39.700,
+      long: -104.973,
+      duration: 'Could be a min and max input from the user -- ex. 15 min 120 max',
+      notes: 'If the weather is poor, meet here instead',
+      max_players: 10,
+      players_going: 5,
+      skill_level: 'Professional',
+      equipment: [
+        {
+          name: 'Net',
+          still_needed: false
+        },
+        {
+          name: 'Ball',
+          still_needed: true
+        }
+      ]
+    },
+    {
+      id: 2,
+      date: 20200409,
+      time: 1700,
+      unix_time: 1586451600,
+      activity: 'Spikeball',
+      location: 'Cheeseman Park',
+      lat: 39.731,
+      long: -104.973,
+      duration: 'Could be a min and max input from the user -- ex. 15 min 120 max',
+      notes: 'If the weather is poor, meet here instead',
+      max_players: 4,
+      players_going: 4,
+      skill_level: 'Casual',
+      equipment: [
+        {
+          name: 'Net',
+          still_needed: true
+        },
+        {
+          name: 'Ball',
+          still_needed: true
+        }
+      ]
+    },
+    {
+      id: 3,
+      date: 20200411,
+      time: 2000,
+      unix_time: 1586635200,
+      activity: 'Kick Ball',
+      location: 'Pepsi Center',
+      lat: 39.748,
+      long: -105.009,
+      duration: 'Could be a min and max input from the user -- ex. 15 min 120 max',
+      notes: 'If the weather is poor, meet here instead',
+      max_players: 20,
+      players_going: 10,
+      skill_level: 'Amateur',
```

Comment:
We should also include a "just for fun" type in the skill level selection.

Reply:
Yeah for sure! I think limiting these inputs to a drop-down when they create an event will definitely limit how wide of a range they can choose when making events